moqt-server/src/lib.rs

```diff
@@ -268,7 +268,9 @@ async fn handle_connection(
                 let pubsub_relation_tx = pubsub_relation_tx.clone();
                 let send_stream_tx = send_stream_tx.clone();
                 let close_connection_tx = close_connection_tx.clone();
+                let object_cache_tx = object_cache_tx.clone();
                 let client= client.clone();
+                let open_subscription_txes = open_subscription_txes.clone();
```

Comment:
I did not know what this txes meant. I checked previous PRs and this was a HashMap, not an instance of mpsc. You need to change the name to something appropriate.

Reply:
I totally forgot to add `downstream_`.
I handled it.